validity/bin/validity.rs

```diff
@@ -53,6 +53,7 @@ async fn main() -> Result<()> {
         agg_proof_mode: env_config.agg_proof_mode,
         submission_interval: env_config.submission_interval,
         mock: env_config.mock,
+        prover_address: env_config.private_key.address(),
```

Comment:
Can you allow configuring this in `read_proposer_env`? Can you add a `PROVER_ADDRESS` environment variable and document it?